cv32e40p/sim/ExternalRepos.mk

```diff
@@ -15,7 +15,7 @@ export SHELL = /bin/bash
 
 CV_CORE_REPO   ?= https://github.com/openhwgroup/cv32e40p
 CV_CORE_BRANCH ?= dev
-CV_CORE_HASH   ?= 1d0ec8d83091bda8e65c36c48614a41065f5fc10
+CV_CORE_HASH   ?= head
```

Comment:
Dangerous, unless you do not expect the RTL to not change.

Reply:
noted. I will take note to change to hash number once the rtl is freeze on our side.